GVFS/GVFS.Platform.Windows/DiskLayoutUpgrades/DiskLayout12to13Upgrade_FolderPlaceholder.cs

```diff
@@ -81,30 +81,32 @@ private static IEnumerable<string> GetFolderPlaceholdersFromDisk(ITracer tracer,
                 {
                     if (!directory.EndsWith(Path.DirectorySeparatorChar + GVFSConstants.DotGit.Root))
                     {
-                        OnDiskFileState fileState = OnDiskFileState.Full;
-                        HResult result = Utils.GetOnDiskFileState(directory, out fileState);
-                        if (result == HResult.Ok)
+                        if (Directory.Exists(directory))
                         {
-                            if (IsPlaceholder(fileState))
+                            OnDiskFileState fileState = OnDiskFileState.Full;
+                            bool result = Utils.TryGetOnDiskFileState(directory, out fileState);
+                            if (result == true)

```

Comment:
Get rid of the `result` variable and change to 
`if (Utils.TryGetOnDiskFileState(directory, out fileState))`